zeeguu_api/api/sessions.py

```diff
@@ -20,14 +23,14 @@ def add_user(email):
     password = request.form.get("password")
     username = request.form.get("username")
     if password is None:
-        flask.abort(400)
+        return make_error(400, "Invalid value")
```

Comment:
should this not be something more precise?

Reply:
I added a test for password length here now. I noticed it was still missing. The message now reads "Password should be at least 4 characters long" if no password is given or too short.